### PR TITLE
Update docs to note changes in --mane --mane_select flags (e113)

### PR DESCRIPTION
**Pull request by nakib103**
sandbox url - http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/script/vep_options.html#opt_mane

### CODE REVIEW

**Comment by veidenberg** [comment]
@nakib103 is this PR still relevant for e112?

**Comment by jamie-m-a** [comment]
Hi @veidenberg Nakib is off until next week - he should be able to respond on Monday.

File: docs/htdocs/info/docs/tools/vep/script/vep_options.html [hunk 1]
```diff
@@ -1327,15 +1327,16 @@ <h3>Note</h3>
       <td>&nbsp;</td>
       <td>
         Adds a flag indicating if the transcript is the <a href="/info/genome/genebuild/mane.html">MANE Select</a> or <a href="/info/genome/genebuild/mane.html">MANE Plus Clinical</a> 
-        transcript for the gene. <i>Not used by default</i>
+        transcript for the gene. If <a href="#opt_cache">--cache</a> or <a href="#opt_database">--database</a> annotation source is used the alternative stable id is also added.
```
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_options.html:1330]:
```suggestion
        transcript for the gene. If <a href="#opt_cache">--cache</a> or <a href="#opt_database">--database</a> annotation source is used, the alternative transcript stable ID is also added. 
```

File: docs/htdocs/info/docs/tools/vep/script/vep_options.html [hunk 2]
```diff
@@ -1346,15 +1347,16 @@ <h3>Note</h3>
       <td>&nbsp;</td>
       <td>
         Adds a flag indicating if the transcript is the <a href="/info/genome/genebuild/mane.html">MANE Select</a>
-        transcript for the gene. <i>Not used by default</i>
+        transcript for the gene. If <a href="#opt_cache">--cache</a> or <a href="#opt_database">--database</a> annotation source is used the alternative stable id is also added.
```
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_options.html:1350]:
```suggestion
        transcript for the gene. If <a href="#opt_cache">--cache</a> or <a href="#opt_database">--database</a> annotation source is used, the alternative transcript stable ID is also added. 
```

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]
Hey @nakib103, thanks for the update. I left a couple of small suggestions.

**Review by nuno-agostinho — APPROVED** [review]
Looks good to me, thanks @nakib103!

**Pull request merged**